CHANGES.md

```diff
@@ -80,4 +80,4 @@ The following parameters have been removed from the DC/OS installer:
 
 ### Fixed and improved
 
-* Fix preflight docker version check failing for docker 1.19. (DCOS-56831)
\ No newline at end of file
+* Fix preflight docker version check failing for docker 1.19. (DCOS-56831)
```

Comment:
I don't know why this changed.

Reply:
When you edited the Changes.md file you added a new line to the end of the file.

Reply:
I removed the new line and the change still appears. It also doesn't look like a new line change.

Reply:
Thanks for discussing. Also not sure, but no worries.